.gitignore

```diff
@@ -51,25 +51,66 @@ npm-debug.log.*
 !/img/preston-login-wink@2x.png
 !/img/preston-login@2x.png
 !/img/questionmark.png
-!/img/module/default.png
-!/img/genders/index.php
-!/img/admin/index.php
+!/img/admin/
+!/img/c/
+/img/c/*
 !/img/c/index.php
+!/img/cms/
+/img/cms/*
 !/img/cms/index.php
+!/img/cms/cms-img.jpg
+!/img/co/
+/img/co/*
 !/img/co/index.php
+!/img/e/
+/img/e/*
 !/img/e/index.php
-!/img/jquery-ui
+!/img/flags/
```

Comment:
```suggestion
!/img/flags/
/img/flags/*
```

Reply:
If we follow the same logic as other folders, it should be this way no?

Reply:
@jolelievre Yes, is a same logic, but flags is already committed, I think is useless to do this, some opinions ?